networkx/algorithms/bipartite/tests/test_centrality.py

```diff
@@ -14,6 +14,9 @@ def setup_class(cls):
         cls.top_nodes = [
             n for n, d in cls.davis.nodes(data=True) if d["bipartite"] == 0
         ]
+        G = nx.complete_bipartite_graph(3, 3)
+        G.add_edge(6, 7)
+        cls.disconnected = G
```

Comment:
+1 for adding this test case. However, since it's a *new* case I'd vote to decouple from the parent class and simply add it as it's own test function, i.e. something like:

```python

def test_bipartite_closeness_centrality_unconnected():
    G = nx.complete_bipartite_graph(3, 3)
    G.add_edge(6, 7)
    ... rest of the test here
```

Reply:
> 

Sounds good Ill decompose these into one PR per file and add comments on deletions/changes to live code